base/.env

```diff
@@ -22,6 +22,7 @@ NDSSCPIMEGA_VERSION=1.0.0
 LIBSSCPIMEGA_VERSION=fb8acf533a7c01b5266bf32d60d1a5f923e19523
 
 MOTOR_VERSION=R7-3-1
+PIGCS2_VERSION=60af8bdb17c1717e4545d8170f820e358ce31458
```

Comment:
Do you think we could ask someone to add a tag to the module? It's okay to have the commit, but a tag is usually better :)

Reply:
I will talk to the person responsible about the changes.